Update migrate.md (#1211)
withastro/docs: src/pages/en/migrate.md

2. Change your existing `.md` file extensions to `.mdx`

> 3. Remove the `setup:` property from your frontmatter, and write its [ECMAScript Module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) import statements below the frontmatter.

Reviewer: ```suggestion
3. Remove any `setup:` properties from your frontmatter, and write any import statements below the frontmatter instead.
```
I don't find the longer name particularly useful, when the link doesn't refer to it or explain what that phrase means. And, I don't think the link to import is helpful because we don't have to tell people what an import is. In the context of these instructions, you are simply writing the exact same statement in a different place. The link is only distracting, I think? You typically don't want to take readers away from a page unless it necessary.

I did change the text to be more clear about taking the import statement that is already there and writing it below the code fence instead.
